### PR TITLE
Decrease font-size of .lead and .display-N on XS screens

### PR DESCRIPTION
**Pull request by Mevrael**
Such big fonts on portrait phones unacceptable. Totally unreadable and break layout.

### CODE REVIEW

**Comment by cvrebert** [comment]
I would guess this will be rejected per #17095.

File: scss/_type.scss [hunk 1]
```diff
@@ -148,3 +148,30 @@ mark,
     content: "\00A0 \2014"; // nbsp, em dash
   }
 }
+
+@include media-breakpoint-down(xs) {
+  // decrease font size on portrait phones and smallest devices
+
+  .lead {
+    font-size: $lead-font-size / 1.1;
+    font-weight: $lead-font-weight;
```
Comment (cvrebert) [inline scss/_type.scss]: Repeating the font-weight declarations seems unnecessary.

**Comment by Mevrael** [comment]
It's not changing base font, only classes with biggest font-sizes which makes them totally unreadable. Just open a default jumbotron example on mobile portrait http://v4-alpha.getbootstrap.com/examples/jumbotron/ and imagine now a bit more realistic slogan. Also when you have a long word all your layout will be broken.

However, I have 2nd option I would recommend more, we could add separate file for mobile typography like _type_mobile.scss.

**Comment by patrickhlauke** [comment]
FWIW i agree that on smaller viewport sizes, some of the bigger elements (e.g. `h1`) could really do with being slightly smaller (it's one of the first things I override when making my own Bootstrap-based sites).

**Comment by mdo** [comment]
Going to punt on this and revisit some of this with a more holistic view than just the utilities. Thanks though!

**Pull request closed without merging**